recipes-graphics/xorg-xserver/xserver-xf86-config/xenclient-uivm/xorg.conf

```diff
@@ -0,0 +1,20 @@
+# Xorg configuration for UIVM.
+
+Section "ServerFlags"
+        Option "DontZap"        "true"
+        Option "DontVTSwitch"   "true"
+        Option "BlankTime"      "0"
+EndSection
+
+Section "Device"
+        Identifier "fbdev0"
+        Driver     "fbdev"
+        Option     "ShadowFB"   "false"
+EndSection
+
+Section "Screen"
```

Comment:
Looking at my WIP branch, I don't have a Screen section.  I think it is created automatically, so that could be dropped.

Reply:
Oh nice, I could not find if the `DefaultDepth` was always set at 24, although it seems to be the case everywhere.

Reply:
... Sorry I spoke too soon. There is definitely a slowdown without `DefaultDepth 24`, which leads me to believe this is not the default value. [The documentation](https://www.x.org/releases/X11R7.7/doc/man/man5/xorg.conf.5.xhtml) mentions that the default is usually 8.